http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/util/AKSManualTest.scala

```diff
@@ -1,214 +0,0 @@
-package org.broadinstitute.dsde.workbench.leonardo.util
```

Comment:
I just removed this file. It was useful for testing apps early on, but I don't think it works anymore. It's in git history if we ever need it again.